### PR TITLE
[WFCORE-6527]: Upgrade JGit from 6.6.0.202305301015-r…

### PR DESCRIPTION
**Pull request by ehsavoie**
… to 6.7.0.202309050840-r

* Upgrading jgit version:https://projects.eclipse.org/projects/technology.jgit/releases/6.7.0
* Explicit dependency on common-codec

Issue: https://issues.redhat.com/browse/WFCORE-6527

### CODE REVIEW

**Comment by yersan** [comment]
supersedes https://github.com/wildfly/wildfly-core/pull/5650

File: pom.xml [hunk 1]
```diff
@@ -722,6 +723,11 @@
                 <artifactId>commons-cli</artifactId>
                 <version>${version.commons-cli}</version>
             </dependency>
+            <dependency>
+                <groupId>commons-codec</groupId>
+                <artifactId>commons-codec</artifactId>
+                <version>${version.commons-codec}</version>
+            </dependency>
```
Comment (yersan) [inline pom.xml:730]: Is this necessary? This comment makes it a bit reluctant https://github.com/wildfly/wildfly-core/pull/4133/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R800

Comment (ehsavoie) [reply pom.xml:730]: @darranl could you take a look ? Is elytron the one which should be bringing in common-codec ?

Comment (ehsavoie) [reply pom.xml:730]: @darranl @fjuma  any input ?

**Comment by darranl** [comment]
@ehsavoie FYI please use the request review feature on PRs, it gives us an easier way to see which PRs are waiting on us if we miss pings.

**Comment by bstansberry** [comment]
@ehsavoie Please change the commit message to reference WFCORE-6527, as per the discussion we had I used WFCORE-6484 for upgrade to 6.6.1

**Comment by ehsavoie** [comment]
superseded by https://github.com/wildfly/wildfly-core/pull/5688

**Pull request closed without merging**